src/data/waypoint.json

```diff
@@ -7,7 +7,8 @@
     "searchOnlyApiKey": "5e4adfd8094367056501547d6fedb6c5"
   },
   "analyticsConfig": {
-    "includedDomains": "waypointproject.io www.waypointproject.io"
+    "includedDomains": "waypointproject.io www.waypointproject.io",
+    "segmentWriteKey": "9mlIVayJbNtJW2EOdAFKHNKcdLAgEDlV"
```

Comment:
we're declaring this in waypoint's repo right now, and it's used on the client to load a script, so there should be no risk having this here.

ref: https://github.com/hashicorp/waypoint/blob/main/website/next.config.js#L41